install_files/ansible-base/build-deb-pkgs.yml

```diff
@@ -1,27 +1,4 @@
 ---
-# Ensure that the build machine is present. If it hasn't been created,
-# then Vagrant will not add it to the generated inventory file. Since
-# the build machine is required for staging runs, error out with a helpful
-# message, pointing the developer to the docs.
-- name: Ensure Build VM is available.
-  hosts: localhost
-  connection: local
-  gather_facts: no
-  become: no
-  tasks:
-    - name: Check for "build" machine in inventory.
-      fail:
-        msg: >-
-          The "build" VM is not reachable, but is required for building the
-          app code Debian packages on the staging machines. Run `vagrant up build`,
-          then `vagrant provision /staging/`. Thereafter you can run
-          `ANSIBLE_ARGS="--tags rebuild" vagrant provision /staging/` to rebuild
-          and reinstall the application code.
-      when: "'build' not in groups.all"
-  tags:
-    - rebuild
-    - local_build
-
 # All plays in this playbook target the build host, but most be invoked separately,
```

Comment:
Please also remove the comment starting on L25, it's no longer accurate and should be deleted.